MAINTAINERS

```diff
@@ -102,6 +102,13 @@ R:	Peng Fan <peng.fan@nxp.com> [@MrVan]
 S:	Maintained
 F:	core/arch/arm/plat-imx/
 
+NXP (Freescale) i.MX family
+R:	Cedric Neveux <cedric.neveux@nxp.com>
```

Comment:
Consider adding Github id as is done for many other platforms.

Reply:
ok, I'll do it